### DISCUSSION TITLE
I'd rather sing Elvis songs!

### DISCUSSION THREAD

**Discussion by TheAustralianWritingAcademy** [vercel/next.js#29346] [Help]
No this isn't spam, but rather offering deserved kudos to all in this community. So after paying a developer to make a website (which I am pleased with), I ended up with an 'incomplete project' (not his fault, he got busy). I am faced with four little 'things' I am wrestling with as a non-webmaker guy:

1. Editable text fields on something called Contentful. 
2. A bunch of code which I can't understand on Github (where the images are uploaded). 
3. Something called Vercel which spits up 'Deployment Errors'.
4. And zero experience in HTML and all other associated jargon - you see, my work is mainly in Keynote and Pages.

How could it be THIS hard? I have to laugh, cos I just wanted to change a couple background images. I took his advice and uploaded a replacement image in Github and named it exactly the same as the image to be replaced - which I deleted first to avoid any naming issues. Enter Vercel, the gatekeeper, who says "Thou shall not deploy" or some kind of error like that. And that's it. For something as simple as change an image, no-can-do.

My question is, why do people work in this complicated sphere when there are tools that are basically click, paste, enjoy? A bit like Keynote and Pages in fact.

You guys are geniuses and I admire your intellect, skills and persistence. I am definitely feeling like a Hound Dog left with making a new website in MailChimp of all places!!! Click... paste... enjoy. Come visit me at Heartbreak Hotel :(

PS My post comes from an utterly 'respect-for-you-guys' tone. Please don't misread any disrespect as I'm too old to entertain a lively debate. I've spent 2 days to try and change a picture. I just... don't get it. Why do things this way? Why have 3 'web places' (Contentful, Github, Vercel) to try and make 1 website. It seems bizarre. I know there'll be a good reason. I just can't see it due to my lack of knowledge.

**Comment by MathiasKandelborg** [comment]
Do you still need help? I might be able to direct you to what you need to do in order to avoid the ominous 'deployment error' and get on with changing that picture.

- **Contentful** is a specialized place to have 'content', its whole job is to make it easy and practical to have all content there.
- **Github** is the place where the website resides. Also, some of the static content e.g. background pictures would be nice to have here. It's essentially a place for your website files and static files to live (free of charge at that). Think GoDaddy hosting or the likes, but modern and 10x better, although it's not always easier for the uninitialized.
- **Vercel** is where the website is hosted. The company also makes the framework - which I'm guessing your developer used - Next.js. Which is incredibly good, and used by companies such as Netflix (go look at their showcase, it's really impressive).

It might be confusing to have 3 different 'things', but they are all specialized in what they do, meaning the cost savings, existing infrastructure, and other benefits you get from each 'thing' accumulate to a very low maintenance website with low cost and great performance across the world.

Let me know and I'll take a look. I like helping and this seems like a problem that could be fixed very easily.
